web/api/queries/countif/README.md

```diff
@@ -0,0 +1,34 @@
+<!--
+title: "CountIf"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/web/api/queries/countif/README.md
+-->
+
+# CountIf
+
+> This query is available as `countif`.
+
+CountIf returns the percentage of points in the database that satisfy the condition supplied.
+
+The following conditions are available:
+
+- `!` or `!=` or `<>`, different than
+- `=` or `:`, equal to
+- `>`, greater than
+- `<`, less than
+- `>=`, greater or equal to
+- `<=`, less or equal to
+
+The target number and the desired condition can be set using the `group_options` query parameter, as a string, like in these examples:
+
+- `!0`, to match any number except zero.
+- `>=-3` to match any number bigger or equal to -3.
+
+## how to use
+
+This query cannot be used in alarms.
+
+`countif` changes the units of charts. The result of the calculation is always from zero to 1, expressing the percentage of database points that matched the condition. 
+
+In APIs and badges can be used like this: `&group=countif&group_options=>10` in the URL.
```

Comment:
What I described [here](https://github.com/netdata/netdata/pull/13155/files#r901202168) also happens when I add `group_options`, but I have another concern. 

Firstly I repeat the test given as argument `group_options=>10`:

```sh
$ curl -k https://localhost:19999/api/v1/data?chart=filesystem.vfs_deleted_objects&group=countif&group_options=%3E10
{
 "labels": ["time","delete"],
    "data":
 [
 [ 1655690840,null],
 [ 1655690835,0],
 [ 1655690830,0],
 [ 1655690825,0],
 [ 1655690820,0],
 [ 1655690815,0],
 [ 1655690810,0],
 [ 1655690805,0],
 [ 1655690800,0],
 [ 1655690795,0],
```

After this I modified for a value that does not make any sense `group_options=><=10`:

```sh
$ curl -k https://localhost:19999/api/v1/data?chart=filesystem.vfs_deleted_objects&group=countif&group_options=%3E%3C=10
{
 "labels": ["time","delete"],
    "data":
 [
 [ 1655691000,null],
 [ 1655690995,100],
 [ 1655690990,100],
 [ 1655690985,100],
 [ 1655690980,100],
 [ 1655690975,100],
 [ 1655690970,100],
 [ 1655690965,100],

[ 1655690940,100],
 [ 1655690935,100],
 [ 1655690930,100],
 [ 1655690925,100],
 [ 1655690920,100],
 [ 1655690915,100],
```

For both queries I had the same `200` as answer, but a complete different output:

```sh
2022-06-20 02:07:00: 112872: 21375 '[localhost]:51442' 'DATA' (sent/all = 348/2230 bytes -84%, prep/sent/total = 0.07/0.10/0.17 ms) 200 '/api/v1/data?chart=filesystem.vfs_deleted_objects&group=countif&group_options=>10'
0.07/0.11/0.18 ms) 200 '/api/v1/data?chart=filesystem.vfs_deleted_objects&group=countif&group_options=><=10'
```

Why are we not returning any error when a wrong argument is given?